### DISCUSSION TITLE
Cannot read property 'home' of undefined

### DISCUSSION THREAD

**Discussion by infinitypaul** [hammerstonedev/sidecar#113] [Q&A]
I use laravel vapor, everything works well on my local and staging environment, but on switching the env to production , even locally, I keep keeping the error in the image below

<img width="1524" alt="image" src="https://user-images.githubusercontent.com/15332137/203889897-1fb011b6-dc98-4f7d-9576-6e63017d77e6.png">

**Comment by aarondfrancis** [comment]
I'll need some more information than that. Where is `home` being referenced? What code are you deploying?
